### PR TITLE
Translated Privacy Policy to German

### PR DESCRIPTION
**Pull request by jutta-frieden**
Added the German privacy policy, which we received from our lawyers in Germany last Friday.

### CODE REVIEW

**Comment by jutta-frieden** [comment]
@georg911 Would you mind being the second proof reader for the German privacy policy?
@Ken-Lo Feel free to look for any non-language issues.
Thanks both!

File: app/pages/legal/privacy/legal-privacy.de.js [hunk 1]
```diff
@@ -11,164 +11,192 @@ export default class LegalPrivacyDe extends React.Component {
...
-          Where the words '<em>GoCardless</em>', '<em>we</em>', '<em>us</em>' or '<em>our</em>' are
-          used in this document, they are all references to GoCardless Limited.
+          Wenn in diesem Dokument die Begriffe <em>GoCardless</em>, <em>wir</em>, <em>uns</em> oder <em>unser(e)</em> verwendet werden,
+          beziehen sie sich alle auf GoCardless Ltd., 338-346 Goswell Road, London, EC1V 7LQ, Vereinigtes Königreich.
         </p>
         <p className='para'>
-          Your privacy is very important to us. We have put in place measures to ensure that
-          any personal information or data that we obtain from you is processed in accordance
-          with the accepted principles of good information handling.
+         Bitte lesen Sie sich diese Richtlinie aufmerksam durch, bevor Sie diese Webseite benutzen.
```
Comment (Ken-Lo) [inline app/pages/legal/privacy/legal-privacy.de.js]: Very minor... we would normally indent this by two spaces!

File: app/pages/legal/privacy/legal-privacy.de.js [hunk 2]
```diff
@@ -11,164 +11,192 @@ export default class LegalPrivacyDe extends React.Component {
...
+          Wir haben Geschäftsbeziehungen zu Personen, die weder im Zusammenhang mit ihrer gewerblichen noch ihrer selbständigen beruflichen
+          Tätigkeit  mithilfe von GoCardless bezahlen (<em>Verbraucher</em>). Wir haben auch Geschäftsbeziehungen mit Leuten, die GoCardless
+          im Rahmen ihrer unternehmerischen Tätigkeit benutzen, wie z.B. Lieferanten, die Verbrauchern die Zahlung mit GoCardless
+          per Lastschrift ermöglichen möchten (<em>Geschäftskunden</em>).
+          Mit einzelnen Geschäftskunden wurden von uns Verträge abgeschlossen, durch die einige ihrer Rechte modifiziert werden könnten.
         </p>
 
         <h3 className='u-text-heading-light u-color-heading u-margin-Vm u-text-s'>
-          What this privacy policy covers
+        Was regelt diese Datenschutzrichtlinie?
```
Comment (Ken-Lo) [inline app/pages/legal/privacy/legal-privacy.de.js]: Same with indentation

File: app/pages/legal/privacy/legal-privacy.de.js [hunk 3]
```diff
@@ -11,164 +11,192 @@ export default class LegalPrivacyDe extends React.Component {
...
-          Personal information is defined as information that may be used to identify a living
-          individual, such as their title, name, address, email address and phone number.
+          Personenbezogene Daten sind Informationen über persönliche oder wesentliche Umstände einer identifizierten oder
+          identifizierbaren lebenden Person, wie Anrede, Name, Adresse, E-Mail-Adresse, Telefonnummer,
+          bestimmte Arten von Serverprotokollen von Ihrem Browser, Ihre IP-Adresse und bestimmte Arten von Cookie-Informationen.
         </p>
 
         <h3 className='u-text-heading-light u-color-heading u-margin-Vm u-text-s'>
-          Information collection and use
+        Erfassung und Verwendung von Informationen
```
Comment (Ken-Lo) [inline app/pages/legal/privacy/legal-privacy.de.js]: Indentation

File: app/pages/legal/privacy/legal-privacy.de.js [hunk 4]
```diff
@@ -11,164 +11,192 @@ export default class LegalPrivacyDe extends React.Component {
...
+        <p className='para'>
+          Telefongespräche können aus Sicherheits- und aufsichtsbehördlichen Gründen aufgezeichnet werden und
+          gemäß unseren Qualitätskontrollverfahren mitgehört werden.
+        </p>
+        <p className='para'>
+          Wenn Sie uns Ihren Lebenslauf zusenden und wir für den Fall, dass sich in der Zukunft eine Gelegenheit
+          einer Zusammenarbeit ergibt, Ihre Angaben speichern möchten, werden wir Sie um Ihre Zustimmung bitten.
+        </p>
+        <p className='para'>
+         Der einzige Fall, in dem wir Ihre Angaben für andere Zwecke als diejenigen, die oben angegeben werden,
```
Comment (Ken-Lo) [inline app/pages/legal/privacy/legal-privacy.de.js]: Indentation

File: app/pages/legal/privacy/legal-privacy.de.js [hunk 5]
```diff
@@ -11,164 +11,192 @@ export default class LegalPrivacyDe extends React.Component {
...
+        <p className='para'>
+          Wir können Ihre persönlichen Informationen an unsere Unterauftragnehmer weitergeben,
+          damit sie Dienstleistungen für uns erbringen, aber zu keinem sonstigen Zweck.
+          Wir respektieren Ihre Privatsphäre und werden ohne Ihre ausdrückliche Erlaubnis keine persönlichen
+          Informationen über Sie an andere Personen oder nicht mit uns verbundene Unternehmen vermieten, verkaufen oder weitergeben.
         </p>
-
         <h3 className='u-text-heading-light u-color-heading u-margin-Vm u-text-s'>
-          Information collection and use
+         Erfassung und Verwendung von Informationen
```
Comment (Ken-Lo) [inline app/pages/legal/privacy/legal-privacy.de.js]: Indentation

**Comment by georg911** [comment]
@jutta-frieden Nothing found, all good! After this exciting read I'm having a nap :sleeping: .. ;-)

File: app/pages/legal/privacy/legal-privacy.de.js [hunk 6]
```diff
@@ -11,164 +11,192 @@ export default class LegalPrivacyDe extends React.Component {
...
-          origin and any recipients of it as well as the purpose of any data processing carried out.
-          Please note that, in accordance with the Data Protection Act 1998, a £10 admin fee is
-          applicable. For further information, please contact us by
-          emailing <Href to='email'
-          className='u-link-color-p u-text-underline' itemProp='email' /> with
-          the subject 'Data subject access request'.
+          Sie können eine Kopie der personenbezogenen Daten, die wir über Sie speichern und verwenden,
+          verlangen und die Herkunft und etwaige Empfänger hiervon sowie den Zweck etwaiger erfolgter Datenbearbeitung von uns erfahren.
+          Für weitere Informationen kontaktieren Sie uns bitte per E-Mail unter <Href to='email'
+          className='u-link-color-p u-text-underline' itemProp='email' /> mit Betreff ‚Auskunftsersuchen einer betroffenen Person‘.
```
Comment (Ken-Lo) [inline app/pages/legal/privacy/legal-privacy.de.js]: Betreff‚ Auskunftsersuchen?

File: app/pages/legal/privacy/legal-privacy.de.js [hunk 7]
```diff
@@ -11,164 +11,192 @@ export default class LegalPrivacyDe extends React.Component {
...
+          oder ihre Löschung von uns zu verlangen. Verwenden Sie die Kontaktangaben am Ende dieser Webseite,
+          um Fragen zu stellen oder die Korrektur, Beschränkung oder Löschung Ihrer personenbezogenen Daten zu verlangen.
         </p>
 
         <h3 className='u-text-heading-light u-color-heading u-margin-Vm u-text-s'>
           Cookies
         </h3>
         <p className='para'>
-          Our web site uses cookies – small text files stored on your computer – in two ways.
+         Ein Cookie ist eine kleine Textdatei, die eine Webseite auf Ihre Festplatte oder Ihren Internetbrowser überträgt,
```
Comment (Ken-Lo) [inline app/pages/legal/privacy/legal-privacy.de.js]: Indentation

**Comment by Ken-Lo** [comment]
:+1: Looks good to me, with indentation fixes

**Pull request merged**